app/views/support/PlanOps.scala

```diff
@@ -126,6 +126,15 @@ object PlanOps {
         case _ => "https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions"
       }
 
+    def termsAndConditionsHref: String =
+      product match {
+        case Delivery => "https://www.theguardian.com/subscriber-direct/subscription-terms-and-conditions"
+        case Voucher => "https://www.theguardian.com/subscriber-direct/subscription-terms-and-conditions"
+        case Product.Digipack => "https://www.theguardian.com/info/2014/aug/06/guardian-observer-digital-subscriptions-terms-conditions"
+        case _: Product.Weekly => "https://www.theguardian.com/info/2014/jul/10/guardian-weekly-print-subscription-services-terms-conditions"
+        case _ => "Sensible default" //TODO
```

Comment:
Product also includes Contributions and Membership, so I'm undecided about what to link in that case. Any ideas?